test/terra/backends/aer_simulator/test_measure.py

```diff
@@ -265,6 +266,75 @@ def test_measure_stablizer_64bit(self, method, device):
 
         self.assertDictAlmostEqual(output, targets, delta=delta * shots)
 
+    @supported_methods(["stabilizer"], [65, 127, 433])
+    def test_measure_sampling_large_stabilizer(self, method, device, num_qubits):
+        """Test sampling measure for large stabilizer circuit"""
+        paulis = qi.PauliList([num_qubits * i for i in ["X", "Y", "Z"]])
+        qc = QuantumCircuit(num_qubits)
+        for pauli in paulis:
+            for i, p in enumerate(pauli):
+                if p == qi.Pauli("Y"):
+                    qc.sdg(i)
+                    qc.h(i)
+                elif p == qi.Pauli("X"):
+                    qc.h(i)
+        qc.measure_all()
+        backend = self.backend(method=method)
+        result = backend.run(
+            transpile(qc, backend, optimization_level=0), shots=10, seed_simulator=12345
+        ).result()
+        counts = result.get_counts()
+
+        ref_counts = {}
+        if num_qubits == 65:
+            ref_counts.update(
+                {
+                    "00000010101011110100110101101101101100010011000110011100101001010": 1,
```

Comment:
Just to test whether stabilizer can work for large-qubit circuits, we can use GHZ and then validate result with more simpler counts.